src/main/java/calculator/Number.java

```diff
@@ -12,6 +12,9 @@ public Number(String number) {
         if (number.equals("")) {
             throw new IllegalArgumentException("숫자가 빈값이면 안됩니다.");
         }
+        if (!number.chars().allMatch(Character::isDigit)) {
+            throw new IllegalArgumentException("숫자로 변환할 수 없습니다.");
+        }
         this.number = Integer.parseInt(number);
```

Comment:
지금과 같은 방법도 좋고 또는 발생하는 에러를 잡아서 처리하는 것도 좋은 방법이 될 수 있을 것 같아요!!

```suggestion
        try {
            this.number = Integer.parseInt(number);
        } catch(NumberFormatException e) {
            throw new IllegalArgumentException("숫자로 변환할 수 없습니다.");
        }
```